src/content/docs/en/recipes.mdx

```diff
@@ -32,3 +32,4 @@ Add your own here! See our [recipes contributing guide](https://github.com/witha
 - [Add an RSS feed to Astro using Storyblok's stories](https://straffesites.com/en/blog/rss-feed-astro-storyblok)
 - [Using tRPC in Astro](https://www.thomasledoux.be/blog/using-trpc-astro-islands-react)
 - [How to make your Astro website multilingual with Crowdin: Astro localization guide](https://crowdin.com/blog/2023/06/21/astro-localization-and-i18n)
+- [Astro RSS Feed Blog Post Images](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
```

Comment:
```suggestion
- [Add blog post images to your Astro RSS Feed](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
```
This doesn't have to be the exact title! Let's give them something so they know what they're clicking on!